CHANGELOG.md

```diff
@@ -23,6 +24,10 @@
 - **Breaking change:** CSV config field `role-path` is now `role-paths` and takes a list of strings. Users can now specify multiple `Role` and `ClusterRole` manifests using `role-paths`. ([#1704](https://github.com/operator-framework/operator-sdk/pull/1704))
 - Make `ready` package idempotent. Now, a user can call `Set()` or `Unset()` to set the operator's readiness without knowing the current state. ([#1761](https://github.com/operator-framework/operator-sdk/pull/1761))
 
+### Deprecated
+
+### Removed
+
```

Comment:
Remove these four lines from the v0.10.0 section.

Reply:
👍